### PR TITLE
PEP 609: Mark as accepted

### PR DESCRIPTION
**Pull request by di**

### CODE REVIEW

File: pep-0609.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Author: Dustin Ingram <di@python.org>,
 Sponsor: Paul Ganssle <paul@ganssle.io>
 BDFL-Delegate: TBD
 Discussions-To: https://discuss.python.org/t/pep-609-pypa-governance/
-Status: Draft
+Status: Accepted
```
Comment (brettcannon) [inline pep-0609.rst:11]:
```suggestion
Status: Active
```

Comment (di) [reply pep-0609.rst]: Does this mean we're already ready to implement the listed "changes would better accommodate the PyPA’s usage of the PEP process" in https://github.com/python/steering-council/issues/30?

**Comment by brainwane** [comment]
For cross-linking: this is thanks to https://github.com/python/steering-council/issues/30#issuecomment-661268123 .

**Comment by brainwane** [comment]
@di Actualy, I have a very mild preference for the commit message mentioning https://github.com/python/steering-council/issues/30#issuecomment-661268123 , or the issue.

**Pull request merged**